device-api/src/arch/mod.rs

```diff
@@ -20,14 +20,14 @@ use crate::DeviceResult;
 pub enum Arch {
     #[enumeration(alias = "Warboy")]
     WarboyB0,
-    Renegade,
+    RNGD,
```

Comment:
seems `Rngd` is correct?

Reply:
Maybe `RNGD` is better. RNGD itself will become a trademark.